_po4a-tools/po4a-create-all-targets.sh

```diff
@@ -71,6 +83,17 @@ process_with_po4a () {
 
         # Get source doc names and set target file names and dirs
         filename=$(basename "$doc" .$ext)
+
+        if [[ $filename == 'general' || $filename == 'navigation' ]] ; then
```

Comment:
Probably not perfect to hardcode the file names but we‘ll have to live with it.

Reply:
I know, I had to do that with the threshold exclusions too... but I can't think of a better way that keeps it simple.